packages/fancy-sitemap/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "fancy-sitemap",
```

Comment:
You should include a main, that way it doesn't have to guess where the entry file is.

Reply:
"main": "./lib"